PolyEngine/RenderingDevice/OpenGL/Src/Proxy/GLTextFieldBufferDeviceProxy.cpp

```diff
@@ -75,5 +75,5 @@ void GLTextFieldBufferDeviceProxy::SetContent(size_t count, const TextFieldLette
 	glBindBuffer(GL_ARRAY_BUFFER, VBO);
 	glBufferData(GL_ARRAY_BUFFER, sizeof(GLfloat) * vboData.GetSize(), vboData.GetData(), GL_DYNAMIC_DRAW);
 	glBindBuffer(GL_ARRAY_BUFFER, 0);
-	CHECK_GL_ERR();
+	// CHECK_GL_ERR(); // spams errors
```

Comment:
Well, usually they are there for a reason. What errors do we get?

Reply:
Legacy, errors related to textures while text draws fine